data-prepper-plugins/translate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/translate/FilePathParser.java

```diff
@@ -0,0 +1,60 @@
+package org.opensearch.dataprepper.plugins.processor.translate;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class FilePathParser {
+    @JsonProperty("mappings")
+    @Valid
```

Comment:
Are these annotations doing anything? This isn't a part of the user configuration right?

Reply:
Yeahh. 
**@JsonProperty** : The file provided will consist of option `mappings`, under which translations will be provided. Since I am using a different variable name `fileMappingConfigs` in the `FilePathPraser.Class`, `@JsonProperty` is required to indicate that `mappings` filed should be mapped to `fileMappingConfigs`

**@Valid** : The type of `fileMappingConfigs` is `List<MappingParameterConfig>`. `MappingParameterConfig.Class` consists of validation checks for the options provided under mappings like `source`, `targets`. To validate those checks we need to provide the `@Valid` annotation.